test-packages/hot-swap-app/README.md

```diff
@@ -0,0 +1,58 @@
+# hot-swap-app
+
+This README outlines the details of collaborating on this Ember application.
```

Comment:
@xg-wang same question here what is the difference between this test app and the basic-app?

If there is a difference and you update the read me there to be clear about what this test-package app is doing and what tests should be added to this package. 

You could use this as an example https://github.com/ember-fastboot/ember-cli-fastboot/tree/master/test-packages

Reply:
Pushed a commit to update the readme